packages/web-app-files/src/components/SideBar/Exif/ExifPanel.vue

```diff
@@ -67,6 +69,7 @@ export default defineComponent({
         return '-'
       }
       if ([5, 6, 7, 8].includes(unref(resource).photo?.orientation)) {
+        // these orientations indicate portrait mode. tika normalizes width and height according to orientation.
```

Comment:
What a weird thing to do, should we maybe handle this on server side instead?

Reply:
please not for this PR 🙈 maybe as a followup 😅

Reply:
sure sure...